ci/conda-env.yml

```
@@ -15,6 +15,7 @@ dependencies:
 - gcsfs
 - geopandas
 - graphviz
+- joblib<1.3  # 1.3 is broken for py36
```

```suggestion
- joblib<1.3; python_version <= '3.6'
```

dunno if its valid syntax in the conda world, but without the marker, people will most likely start running into dephell in their more modern envs